### ISSUE TITLE
VO announces text content and title for all table cells

### ISSUE THREAD

**Issue by dstorey** [dstorey/HTML5accessibility#46]
Seems like a bug as I think title would be accessible description, which isn't usually read out by default, but need to look into it more with other screen readers. (actually I think it should be the accessible name as text contents are probably last :(, so not a bug.)

Not sure way forward, as want title for visual users to make sure they understand icons, but don't want to leave all the cells empty by only using title. 

Current site doesn't use titles.

**Comment by dstorey** [comment]
ChromeVOX doesn't announce both

**Comment by dstorey** [comment]
@stevefaulkner do you know a good way of having a title on the table cells, without it being read out (haven't tested on windows  screen readers yet, but an issue on Mac / VoiceOver).

I have titles so that visual users don't need to check the key to understand symbols in table, but there is text content (hidden via image replacement) in the cells, so VO reads "yes, yes". Feels weird to have empty cells with only a title attribute (or an image for every cell with no alt text)

As mentioned the existing site gets around this by just not having titles at all. I'm suspecting I'm going to have to go this route. Even if Windows screen readers work, it feels wrong to allow Safari to keep reading out things twice.

Current code is basically:

``` HTML
<td title="yes">yes</td>
```

**Comment by dstorey** [comment]
Options that seem less than ideal

``` HTML
  <td>Text Content</td>
  <td title="title">Text Content</td>
  <td title="title"><span role="presentation">Text Content</span></td>
  <td title="title"></td>
  <td aria-label="aria-label" title="title"></td>
  <td aria-label="aria-label" title="title">Text Content</td>
```

Actually turns out none of these except the first one (no title) work in VoiceOver: https://jsfiddle.net/dstorey/uzyLg3cj/

role="presentation" is ignored (and it is kinda pointless to add text for screen readers then ignore it)
Anything without text content reads out "blank" even if there is a title or aria-label
Aria-label with title does override the title (actually most likely makes title the accessible description), but still reads the text content anyway.

Edit: actually I'm being stupid with role="presentation"; getting mixed up with my normal use on img. Should be aria-hidden="true", which is just as verbose, and just as pointless in this case, as it'd be hiding text replacement text that is never visually seen.

**Comment by dstorey** [comment]
Note: VO only announces them when going through cells one by one. If reading full page, it only reads them once.

**Comment by dstorey** [comment]
Temp fixed this by removing titles
